docker/install/ubuntu_install_llvm_from_source.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# This script builds LLVM and clang from the llvm-project tarball
+# using CMake. It is tested with LLVM from version 15.
+
+set -e
+
+LLVM_VERSION_MAJOR=$1
```

Comment:
I agree, and if we remove the $1 parameter, then should we also cleanup the `detect_llvm_version` function? What do you think?

Reply:
We could still use the parameter if we wanted to, similar to:

https://github.com/apache/tvm/blob/6c04ac52bd43065156cabb609f76cd2323449d27/docker/Dockerfile.ci_adreno#L25

Otherwise, yes, `detect_llvm_version` should be removed.